dpnp/dpnp_iface_manipulation.py

```diff
@@ -906,6 +906,14 @@ def asfarray(a, dtype=None, *, device=None, usm_type=None, sycl_queue=None):
 
```

Comment:
We have to remove links on :obj:`dpnp.asfarray` from the docstrings of other functions